Resources/Prototypes/_RMC14/Entities/Clothing/Uniforms/cmb.yml

```
@@ -0,0 +1,17 @@
+- type: entity
+  parent: RMCMarineUniformBase
+  id: RMCSwatCMBUniform
+  name: CMB Riot Control uniform
+  description: A dark set of tactical uniform utilized by the Colonial Marshals, designed to be used by units of riot supression on the distant worlds, under colonial jurisdiction.
```

Small name fix for the name we use here
```suggestion
  description: A dark set of tactical uniform utilized by the Colony Marshals, designed to be used by units of riot supression on the distant worlds, under colony jurisdiction.
```